client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -531,25 +586,66 @@ export class _DisplayTable extends React.Component<
                         </div>
                       )}
                       {searchable && (
-                        <DebouncedTextInput
-                          inputClassName={"search input-sm"}
-                          placeHolder={text_maker("filter_data")}
-                          a11y_label={text_maker("filter_data")}
-                          defaultValue={current_search_input}
-                          updateCallback={(search_value: string) => {
-                            const updated_searches = _.mapValues(
-                              searches,
-                              (value, key) =>
-                                key === column_key ? search_value : value
-                            );
-
-                            this.setState({
-                              searches: updated_searches,
-                              current_page: 0,
-                            });
+                        <div
+                          style={{
+                            height: is_dropdown_filter_applied
+                              ? "6rem"
+                              : "4.5rem",
+                            minWidth: "155px",
                           }}
-                          debounceTime={300}
-                        />
+                          className="input-bar"
+                        >
+                          <div style={{ display: "flex" }}>
+                            <DebouncedTextInput
+                              inputClassName={`search input-sm input-unstyled`}
+                              style={{ width: "100%" }}
+                              placeHolder={text_maker("search_column")}
+                              a11y_label={text_maker("search_column")}
+                              defaultValue={current_search_input}
+                              updateCallback={(search_value: string) => {
+                                const updated_searches = _.mapValues(
+                                  searches,
+                                  (value, key) =>
+                                    key === column_key ? search_value : value
+                                );
+
+                                this.setState({
+                                  searches: updated_searches,
+                                  dropdown_filter: {
+                                    ...dropdown_filter,
+                                    [column_key]: get_default_dropdown_filter(
+                                      col_configs_with_defaults,
+                                      data
+                                    )[column_key],
```

Comment:
Why do we want to reset the state of the filter options when the out dropdown changes? Is that desired behaviour (specifically resetting `active` status)?

Reply:
Hmmm so my thought was that when a user checks off an item, then search for that in the outer search and if the item doesn't show up, that might be confusing.. But now that I think of it, filter being reset on outer search would be unexpected behaviour and would be more confusing. I'll remove this